package/samplers/hill_climb_search/README.md

```diff
@@ -0,0 +1,33 @@
+---
+author: Chinmaya Sahu
+title: Hill Climb Local Search Sampler
+description: This sampler used the Hill Climb Algorithm to improve the searching, by selecting the best neighbors and moving in that direction.
+tags: [sampler, hill climb]
+optuna_versions: [4.0.0]
+license: MIT License
+---
+
+## Abstract
+
+The **hill climbing algorithm** is an optimization technique that iteratively improves a solution by evaluating neighboring solutions in search of a local maximum or minimum. Starting with an initial guess, the algorithm examines nearby "neighbor" solutions, moving to a better neighbor if one is found. This process continues until no improvement is possible, resulting in a locally optimal solution. Hill climbing is efficient and easy to implement but can get stuck in local optima, making it suitable for simple optimization landscapes or applications with limited time constraints. Variants like random restarts and stochastic selection help overcome some limitations.
+
+## Class or Function Names
+
+- HillClimbSearch
+
+## Example
+
+```python
+import optuna
+import optunahub
+   
+def objective(trial):
+    x = trial.suggest_discrete_uniform("x", -10, 10)
+    y = trial.suggest_discrete_uniform("y", -10, 10)
```

Comment:
Since `suggest_discrete_uniform` is deprecated, could you use `suggest_int` or `suggest_float` instead?

```suggestion
    x = trial.suggest_int("x", -10, 10)
    y = trial.suggest_int("y", -10, 10)
```

Reply:
Sure, thanks for notifying me!